corehq/apps/analytics/tasks.py

```diff
@@ -20,6 +20,7 @@
 from django.core.urlresolvers import reverse
 from corehq.util.soft_assert import soft_assert
 from corehq.apps.accounting.models import SoftwarePlanEdition
+from corehq.toggles import deterministic_random
```

Comment:
Nice function. Now that we're using it for more than just a toggle, maybe we should move it somewhere else. I'm not sure where though ... and definitely not blocking.